gammapy/makers/background/reflected.py

```diff
@@ -57,7 +181,7 @@ class ReflectedRegionsFinder(RegionsFinder):
     background estimate for an IACT observation using the reflected regions
     method, see also `~gammapy.makers.ReflectedRegionsBackgroundMaker`
 
-    Parameters
+    Attributes
```

Comment:
We don't use `Attributes` but rather `Parameters` for `__init__` docstring.

Reply:
But this is a class docstring. You either have to use `Attributes` in the class docstring or `Parameters` in the `__init__` docstring. Using `Parameters` in the class docstring is wrong.

Reply:
So far, we have been using the class docstring for the `__init__` docstring, but we have not used `Attributes` at all in the documentation. Our dev doc here is clearly unfinished on the specific point of class docstrings. 
As far as I understand, using `Parameters` in the class docstring is not wrong in general. It is one possible option.  In astropy, you actually find the two forms, but mostly `__init__` `Parameters` in the class docstring.

Reply:
Ok

Reply:
fixed